README.md

```diff
@@ -1,32 +1,60 @@
+    </p>
+</p>
+
+<img alt="Illustraion" src="./docs/workflow.png" style="border-radius: 5px"/>
+
+Composio provides a platform that seamlessly integrates over 200+ apps with your AI agents, enhancing their utility and interactivity. It supports all the features you'll ever need:
+
+- **Compatibility with Major Agent Frameworks**: Autogen, Langchain, CrewAI, Julep, Lyzr, and OpenAI are all compatible with Composio, giving you a simple experience for your workflows.
+- **Full Support for Actions & Triggers**: Composio provides full support for Actions & Triggers, to power your agents with a wide range of functionalities.
+- **Authentication Management**: Composio provides comprehensive management of all authentication types, including `OAuth2`, `OAuth1`, `Basic`, `API_KEY`.
+- **Extensibility**: Don't find a tool you need? Composio is highly extensible, allowing you to add your own set of custom tools.
+- **Secure Environment**: Every user has a secure compute environment, ensuring data protection and privacy.
+
+## Getting started
+ To get started, select the framework you want to use and install the correseponding package:
+- **Autogen**: `pip install composio_autogen`
+- **CrewAI**: `pip install composio_crewai`
+- **Langchain**: `pip install composio_langchain`
+- **Lyzr**: `pip install composio_lyzr`
+- **OpenAI**: `pip install composio_openai`
 
-## Run a single test file
+Login now into your composio account, using the  below CLI command:
+```shell 
+composi-cli login
 ```
-pytest -v -s --log-level=DEBUG tests/autogen_test.py
-```
\ No newline at end of file
+
+<hr/>
+
+### Run a sample use-cases repo for your framework
+The below sample repos are provided to help you get started with Composio. Follow the README.md to start these agents.
+
+| Framework | Use-Case | GitHub Repo | Features Used |
+| --- | --- | --- | --- |
+| Autogen | Sample  | [GitHub Repo](https://github.com/username/repo) | - |
+| CrewAI | Competitor research agent that is triggered with a slack message, starts researching about the said topic and  finally creates a page on notion and reply back to slack. | [GitHub Repo](https://github.com/username/repo) | ![](https://img.shields.io/badge/triggers-8A2BE2) ![](https://img.shields.io/badge/actions-8A2BE2) ![](https://img.shields.io/badge/auth-8A2BE2)
+| Langchain | Notion - Product Discussion Agent: An agent that creates linear isseus fronm the production discussions in notion. | [GitHub Repo](https://github.com/username/repo) | ![](https://img.shields.io/badge/actions-8A2BE2) ![](https://img.shields.io/badge/auth-8A2BE2)
+| Lyzr | Repo Recommender - Lyzer agent that analyzes your previous starred repos and notifies you a similar repo that you might be interested in. | [GitHub Repo](https://github.com/username/repo) | ![](https://img.shields.io/badge/actions-8A2BE2) ![](https://img.shields.io/badge/auth-8A2BE2)
+| OpenAI | Sample Use-Case | [GitHub Repo](https://github.com/username/repo) | - |
```

Comment:
@utkarsh-dixit Can you fix the github links here so we can get this merged?

Reply:
1. https://github.com/joaomdmoura/crewAI
2. https://github.com/langchain-ai/langchain
3. https://github.com/LyzrCore/lyzr